# Description: change the default height so all category are on screen
# Ubuntu: https://bugs.launchpad.net/ubuntu/+source/gnome-control-center/+bug/1728450
# Upstream: https://bugzilla.gnome.org/show_bug.cgi?id=789372
diff --git a/shell/cc-window.ui b/shell/cc-window.ui
index eca04a9..84da945 100644
--- a/shell/cc-window.ui
+++ b/shell/cc-window.ui
@@ -5,7 +5,7 @@
     <property name="can_focus">False</property>
     <property name="window_position">center</property>
     <property name="default-width">980</property>
-    <property name="default-height">640</property>
+    <property name="default-height">770</property>
     <signal name="notify::window" handler="gdk_window_set_cb" object="CcWindow" swapped="no" />
     <signal name="map-event" handler="window_map_event_cb" object="CcWindow" swapped="no" />
     <signal name="key-press-event" handler="window_key_press_event_cb" object="CcWindow" swapped="no" after="yes" />
